extensions/HyperVExtension/src/HyperVExtension/Services/DownloaderService.cs

```diff
@@ -71,4 +129,17 @@ private long GetTotalBytesToReceive(HttpResponseMessage response)
         // We should be able to get the content length from the response headers from the Microsoft servers.
         throw new InvalidOperationException("The content length of the response is not known.");
     }
+
+    public bool IsFileBeingDownloaded(string destinationFilePath)
+    {
+        lock (_lock)
+        {
+            if (_destinationFileDownloadMap.TryGetValue(destinationFilePath, out var subscribers))
```

Comment:
NIT: use `_` instead of `subscribers` to notify readers the variable is unused.

Reply:
updated thanks